keras/applications/mobilenet.py

```diff
@@ -0,0 +1,275 @@
+'''MobileNet models for Keras.
+# Reference
+- [MobileNets: Efficient Convolutional Neural Networks for
+   Mobile Vision Applications](https://arxiv.org/pdf/1704.04861.pdf))
+'''
```

Comment:
Add information about which MobileNet models can be built and what their ImageNet test performance is (and parameter count)